packages/now-python/src/index.ts

```diff
@@ -29,6 +30,27 @@ async function pipenvConvert(cmd: string, srcDir: string) {
   }
 }
 
+async function matchPaths(
+  configPatterns: string | string[] | undefined,
+  workPath: string
+) {
+  if (!configPatterns) {
+    return [];
+  }
+
+  const patterns =
+    typeof configPatterns === 'string' ? [configPatterns] : configPatterns;
+
+  const patternPaths = await Promise.all(
+    patterns.map(async pattern => {
+      const files = await glob(pattern, workPath);
+      return Object.keys(files);
+    })
+  );
+
+  return ([] as string[]).concat(...patternPaths);
```

Comment:
Why not just `return patternPaths`?

Reply:
Good point, I've changed it as you suggested.